src/lib/cfa/hosts.rb

```diff
@@ -0,0 +1,147 @@
+require "yast"
+require "yast2/target_file"
+
+require "cfa/base_model"
+require "cfa/matcher"
+require "cfa/augeas_parser"
+
+module CFA
+  class Hosts < BaseModel
```

Comment:
For the people that did not design CFA::BaseModel, a comment with the full URL of its API docs will be useful: http://www.rubydoc.info/github/config-files-api/config_files_api/CFA/BaseModel